Content.Shared/CCVar/CCVars.cs

```diff
@@ -1159,13 +1159,13 @@ public sealed class CCVars : CVars
         /// How long a client can go without any input before being considered AFK.
         /// </summary>
         public static readonly CVarDef<float> AfkTime =
-            CVarDef.Create("afk.time", 60f, CVar.SERVERONLY);
+            CVarDef.Create("afk.time", 450f, CVar.SERVERONLY);
```

Comment:
This shouldn't be changed, it also controls roletimer afk detection. Should have the "you're going AFK for too long" message be significantly harder to trigger.

Reply:
im aware this also affects that, i just also think its silly for it to count you as being afk after a minute

Reply:
afk-detection was always kind of busted (e.g. it doesnt update when you move your cursor or anything since its serverside) its just immediately obvious with the kick timer now